src/prefab.ts

```diff
@@ -134,7 +134,24 @@ export class Prefab {
     return this.load();
   }
 
-  get configs(): { [key: string]: Config } {
+  extract(): Record<string, Config["value"]> {
+    return Object.entries(this._configs).reduce(
+      (agg, [key, value]) => ({
+        ...agg,
+        [key]: value.value,
+      }),
+      {} as Record<string, Config["value"]>
+    );
+  }
+
+  hydrate(rawValues: RawConfigWithoutTypes | EvaluationPayload): void {
```

Comment:
I'm not sure there is an actual use-case for supporting `EvaluationPayload`. Hoping someone can give me some guidance here with regards to where this particular payload comes from and if we need to support it in the hydration lifecycle.

Reply:
that all makes sense... That said, in the context of this change, we aren't considering them "evaluated" as far as I can tell. What I implement hydrates the cache using the same this.setConfig (now private) method. 

It's also worth calling out that SSR may have evaluated some of the flags in the cache. In that case, I'm assuming you guys already handle flushing the telemetry, and there is no need to pass it to the FE to ensure it's flushed.

Otherwise, I agree there is no need to support the evaluations API here, in which case I will update the hydrate method to explicitly not support that, both in typing + in implementation.

Thoughts?

Reply:
I investigated a bit further based on what comes back from the API when initializing the client as well as how evaluations are tracked and shipped. The stuff in the explicit cache actually doesn't include any of the actual evaluation data. That is tracked in a different data object entirely. So there is no harm in actually supporting the evaluation payload as it exists in the cache.

This raises more questions for me than anything in terms of why this is actually needed in the cache at all. We can have that conversation another day.